sql/catalyst/src/test/scala/org/apache/spark/sql/catalyst/catalog/SessionCatalogSuite.scala

```diff
@@ -536,12 +536,13 @@ abstract class SessionCatalogSuite extends AnalysisTest {
       assert(metadata.viewText.isDefined)
       val view = View(desc = metadata, output = metadata.schema.toAttributes,
         child = CatalystSqlParser.parsePlan(metadata.viewText.get))
+      val alias = AliasIdentifier("view1", Some("db3"))
```

Comment:
instead of creating `AliasIdentifier` manually, we can also define a `SubqueryAlias.apply` which takes db name and table name.

Reply:
Sure, let me do that if this style is preferred. Thanks.